ofctrl/ofctrl_test.go

```diff
@@ -950,13 +950,82 @@ func TestBundle2(t *testing.T) {
 	assert.False(t, found)
 
 	time.Sleep(2 * time.Second)
-	verifyGroup(t, brName, group1, "select", "bucket=bucket_id:50,actions=ct(commit,nat(src=10.0.0.240,random))", true)
+	verifyGroup(t, brName, group1, "select", "bucket=bucket_id:50,actions=ct(commit,nat(src=10.0.0.240,random))", "", true)
 	matchStr := "priority=100,ip,in_port=11"
 	actionStr := "group:2"
 	assert.Truef(t, ofctlDumpFlowMatch(brName, int(ofActor.inputTable.TableId), matchStr, actionStr),
 		"br: %s, target flow not found on OVS, match: %s, actions: %s", brName, matchStr, actionStr)
 }
 
+func TestGroupWithSelectionMethod(t *testing.T) {
+	brName := ovsDriver.OvsBridgeName
+	tcpSrcPortFieldWithValue := openflow15.NewTcpSrcField(0x3f)
+	tcpSrcPortFieldWithoutValue := openflow15.NewTcpSrcField(0xffff)
+	nwProtoFieldWithoutValue := openflow15.NewIpProtoField(0xff)
+	for _, tc := range []struct {
+		name           string
+		groupId        uint32
+		mt             openflow15.NTRSelectionMethodType
+		param          uint64
+		fields         []openflow15.MatchField
+		propertyString string
+	}{
+		{
+			name:           "dp_hash",
+			groupId:        30,
+			mt:             openflow15.NTR_DP_HASH,
+			param:          0x1ffffffff,
+			propertyString: "selection_method=dp_hash,selection_method_param=8589934591",
```

Comment:
What does the value `0x1ffffffff` represent?

Reply:
No special meaning, just use it to test if "method_param" can be set as expected or not.